.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: go
+
+sudo: false
+
+go:
+  - 1.4
+  - 1.5
+  - tip
+
+matrix:
```

Comment:
I'd probably make it fail with Go tip - the code here is simple enough and it would be unusual for this to fail, so it shouldn't be too noisy. Otherwise LGTM.